python/pyspark/context.py

```diff
@@ -1105,6 +1106,18 @@ def getConf(self):
         conf.setAll(self._conf.getAll())
         return conf
 
+    def resources(self):
+        resources = {}
+        jresources = self._jsc.resources()
+        for x in jresources:
+            name = jresources[x].name()
+            jaddresses = jresources[x].addresses()
+            addrs = []
+            for addr in jaddresses:
```

Comment:
a comprehension would be a little cleaner here, e.g. `addrs = [addr for addr in jaddresses]`

Reply:
will update